CI (Windows) just complained:
```
/home/travis/build/LMMS/lmms/include/Track.h:314:24: error: non-static data member initializers only available with -std=c++11 or -std=gnu++11 [-Werror]

  bool m_cursorSetYet = false;
```
Can't we set -std=c++11? Otherwise, I'll change this code ofc.

--- a/include/Track.h
+++ b/include/Track.h
@@ -311,7 +311,7 @@ private:
 	bool m_gradient;
 	int m_mouseHotspotX;
 	int m_mouseHotspotY;
-	bool m_cursorSetYet = false;
+	bool m_cursorSetYet;
 
 	bool m_needsUpdate;
 	inline void setInitialMousePos( QPoint pos )

--- a/include/Track.h
+++ b/include/Track.h
@@ -201,8 +201,8 @@ class TrackContentObjectView : public selectableObject, public ModelView
 	Q_PROPERTY( QColor textShadowColor READ textShadowColor WRITE setTextShadowColor )
 	Q_PROPERTY( QColor BBPatternBackground READ BBPatternBackground WRITE setBBPatternBackground )
 	Q_PROPERTY( bool gradient READ gradient WRITE setGradient )
-	Q_PROPERTY( int mouseHotspotX READ getMouseHotspotX WRITE setMouseHotspotX )
-	Q_PROPERTY( int mouseHotspotY READ getMouseHotspotY WRITE setMouseHotspotY )
+	Q_PROPERTY( int mouseHotspotX WRITE setMouseHotspotX )
+	Q_PROPERTY( int mouseHotspotY WRITE setMouseHotspotY )
 
 public:
 	TrackContentObjectView( TrackContentObject * tco, TrackView * tv );
@@ -228,8 +228,6 @@ public:
 	QColor textShadowColor() const;
 	QColor BBPatternBackground() const;
 	bool gradient() const;
-	int getMouseHotspotX() const { return m_mouseHotspotX; }
-	int getMouseHotspotY() const { return m_mouseHotspotY; }
 	void setMutedColor( const QColor & c );
 	void setMutedBackgroundColor( const QColor & c );
 	void setSelectedColor( const QColor & c );

--- a/src/core/Track.cpp
+++ b/src/core/Track.cpp
@@ -257,8 +257,9 @@ TrackContentObjectView::TrackContentObjectView( TrackContentObject * tco,
 	m_textShadowColor( 0, 0, 0 ),
 	m_BBPatternBackground( 0, 0, 0 ),
 	m_gradient( true ),
-	m_mouseHotspotX(0),
-	m_mouseHotspotY(0),
+	m_mouseHotspotX( 0 ),
+	m_mouseHotspotY( 0 ),
+	m_cursorSetYet( false ),
 	m_needsUpdate( true )
 {
 	if( s_textFloat == NULL )

